plugin.cpp

```diff
@@ -144,9 +144,9 @@ int slurm_spank_local_user_init(spank_t sp, int ac, char **av) {
   if (!args.file) {
     // ensure that --uenv-mount is not present, getopt is called here
     // because it is not allowed to be called in `spank_init`.
-    char** tmp{nullptr};
-    spank_err_t ret = spank_option_getopt(sp, &mount_point_arg, tmp);
-    if (ret != ESPANK_ERROR) {
+    char* buf;
+    spank_err_t ret = spank_option_getopt(sp, &mount_point_arg, &buf);
```

Comment:
Instead of making another call to `spank_option_getop`, we could make `args.mount_point` a `std::optional<string>`, like `args.file`. Or, optionally use the current approach and have an additional boolean value in `args` that indicates whether the value for the mount_point is default or user-provided.
If you use the first approach, a helper member like `args.mount_value_or_default()` could be added to `args` that would return the default value or user provided value.